### PR TITLE
Feature/30162 - Edit 'paragraph' component

### PR DESCRIPTION
**Pull request by kjugi**
+ Add view for styled paragraphs in component

https://dev.snowdog.pro/issues/30162

### CODE REVIEW

File: components/02-elements/paragraph/_paragraph.scss [hunk 1]
```diff
@@ -19,3 +19,17 @@ a {
     color: $color-main1;
     text-decoration: underline;
 }
+
```
Comment (Igloczek) [inline components/02-elements/paragraph/_paragraph.scss]: Wow... I missed this code somehow. Is shouldn't be added just like that in the global scope, b/c will affect the whole page.

Comment (Igloczek) [inline components/02-elements/paragraph/_paragraph.scss]: A component always have to be some class.

Comment (kjugi) [inline components/02-elements/paragraph/_paragraph.scss]: changed to BEM + change html

Comment (kjugi) [inline components/02-elements/paragraph/_paragraph.scss]: changed

File: components/02-elements/paragraph/paragraph.hbs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        <a href="#" class="paragraph--link">
+            {{basicText}}
+        </a>
+    </p>
+
+    <span class="paragraph--text-hook">
+        {{basicText}}
+    </span>
+
+    <pre class="paragraph--preformatted-text">
```
Comment (Igloczek) [inline components/02-elements/paragraph/paragraph.hbs]: Useless clasess

**Comment by Igloczek** [comment]
Test are not passing.

File: components/02-elements/paragraph/paragraph.hbs (hunk 2 above)
Comment (kjugi) [inline components/02-elements/paragraph/paragraph.hbs]: changed

**Pull request merged**